jdi-light-vuetify-tests/src/main/java/io/github/com/custom/cards/CookingHoverCard.java

```diff
@@ -0,0 +1,33 @@
+package io.github.com.custom.cards;
```

Comment:
Please, disable PaginationTests that fail and make runs. They fail every time unfortunately :(